python/mxnet/gluon/estimator/estimator.py

```diff
@@ -77,6 +77,10 @@ def __init__(self, net,
                 raise ValueError("metrics must be a Metric or a list of Metric, "
                                  "refer to mxnet.metric.EvalMetric:{}".format(metrics))
 
+        # Use default mx.metric.Accuracy() for gluon.loss.SoftmaxCrossEntropyLoss()
+        if not self.train_metrics and any([isinstance(l, gluon.loss.SoftmaxCrossEntropyLoss) for l in self.loss]):
```

Comment:
lets get this from a map of Loss->[default metrics] in the next version.

Reply:
Yes, tracking it using JIRA issue: https://issues.apache.org/jira/browse/MXNET-1364